DANS-KNAW/easy-deposit
# EASY-1148, delete fetched items from fetch.txt ->

fixes EASY-1148

#### When applied it will
* Now deleting all items that were fetched to the bag from fetch.txt. If fetch.txt is empty
after that, it is deleted.

#### Where should the reviewer @DANS-KNAW/easy start?
At [the new function](https://github.com/DANS-KNAW/easy-sword2/compare/master...janvanmansum:EASY-1148?expand=1#diff-ad17b49d50f1b93808fb326e8f79d0d8R277)

#### How should this be manually tested?
Use the fetch-example from `easy-sword2-dans-examples`, deposit is and verify that the fetch.txt is deleted, or run it with the examples themselves and user mendeltest and see that archiving to archival storage does not fail any longer.

- [x] Finish refactoring getBagFromDir so that the new version, returning a Try is used everywhere
- [x] Test on `deasy`
